frontend/frontend-sdk/tests/lib/client/PasscodeClient.spec.ts

```diff
@@ -131,8 +131,9 @@ describe("PasscodeClient.initialize()", () => {
 
   it.each`
     status | error
-    ${401} | ${"Unauthorized error"}
+    ${401} | ${"Technical error"}
```

Comment:
Why is this changed to Technical error ?

Reply:
You can always initialise a passcode without having a session, so we mapped a 401 to a general Technical Error. It would be misleading for a user if an Unauthorized error occurs - the message would be something like "You need to log in again".